### PR TITLE
Merge of the Release Note V2.1.0

### PR DESCRIPTION
**Pull request by ctauzin-ads**
New System Release Note of Reference System 2.1.0 with the update of the V1.15 of production-common.

### CODE REVIEW

File: releases/V2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+N/A
+
+#### Bugs closed in this version
+  
+N/A
+
+### processing-sentinel-2
+
+#### Development done by User Story (US)
+
```
Comment (Woljtek) [inline releases/V2.1.0.md:124]: Missing N/A

File: releases/V2.1.0.md [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+[1060 - [RS Core INGESTION] Implement CADIP as a new additional interface](https://github.com/COPRS/rs-issues/issues/1060)  
+[1061 - [RS Core INGESTION] Provide a ingestion-cadip configurations example](https://github.com/COPRS/rs-issues/issues/1061)  
+[1068 - [RS Core DISTRIBUTION] Add new OriginDate attribute to prip index](https://github.com/COPRS/rs-issues/issues/1068)  
+[1069 - [PRIP] Support “and”, “or”, “not”, “in” operators](https://github.com/COPRS/rs-issues/issues/1069)  
+[1070 - [RS Core DISTRIBUTION] Add GeoFootprint properties](https://github.com/COPRS/rs-issues/issues/1070)  
+[1078 - [PRIP] Update entity data model](https://github.com/COPRS/rs-issues/issues/1078)  
+
```
Comment (Woljtek) [inline releases/V2.1.0.md:85]: Remove one line

File: releases/V2.1.0.md [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+| processing-sentinel-2 | [1.8.0](https://github.com/COPRS/processing-sentinel-2/releases/tag/1.8.0) |
+| processing-sentinel-3 | [1.14.0](https://github.com/COPRS/processing-sentinel-3/releases/tag/1.14.0) |
+
+# Status of the software configuration item
+
+## Evolution since previous version
+
+### Functionnal evolutions
+  
+Complete production common V1.15.0
```
Comment (Woljtek) [inline releases/V2.1.0.md:63]: Replace this line by:
Implement CADIP ingestion component
Update distribution components to be compliant with Production Interface Delivery Point_ICD_v1.9 (ESA-EOPG-EOPGC-IF-3)

File: releases/V2.1.0.md [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+[991 - [BUG] UWC - "ProductType+contains" queries returns no result if uppercase](https://github.com/COPRS/rs-issues/issues/991)  
+[961 - [BUG] [OPS] Impossible to use safescale cluster node delete](https://github.com/COPRS/rs-issues/issues/961)  
+[871 - [BUG] End-points "native-api", "scdf', "thanos" is not reachable from Black Box Exporter.](https://github.com/COPRS/rs-issues/issues/871)  
+[615 - [BUG] compression-part1-message-filter loops on error ErrorCallbackNotImplemented.](https://github.com/COPRS/rs-issues/issues/615)  
+[816 - [BUG] Missing S3-SM2-HY/LI/SI images on artifactory](https://github.com/COPRS/rs-issues/issues/816)  
+[927 - [BUG] [S2] IPF error : "No Antimerian segment found in two-parts Geo footprint"](https://github.com/COPRS/rs-issues/issues/927)  
+
+## Advice for use of the software configuration item
+  
+NC
```
Comment (Woljtek) [inline releases/V2.1.0.md:687]: Non Compliant ??? I would expect "N/A -> Non Applicable"

**Review by Woljtek — CHANGES_REQUESTED** [review]

File: releases/V2.1.0.md (hunk 4 above)
Comment (ctauzin-ads) [reply releases/V2.1.0.md:687]: I didn't change this one. It was in the original document, I just copied/pasted this line, but I change it in the new version.

**Review by Woljtek — APPROVED** [review]

**Pull request merged**